Landing Page body done

d-shah1 opened the pull request.

d-shah1 committed:
Landing Page body done

AyushBherwani1998 commented:
Hey @d-shah1 thanks for the  contribution, can you please attach a screenshot for the UI changes your PR is making? It will be helpful for us to review your PR.

d-shah1 committed:
Screenshot of the landing page

This is the screenshot of the landing page and sufficient space for navigation bar is also provided.
